src/sidebar/components/annotation-share-info.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+const propTypes = require('prop-types');
+const { createElement } = require('preact');
+
+const { withServices } = require('../util/service-context');
+
+const SvgIcon = require('./svg-icon');
+
+/**
+ * Render information about what group an annotation is in and
+ * whether it is private to the current user (only me)
+ */
+function AnnotationShareInfo({ annotation, groups, isPrivate }) {
+  const group = groups.get(annotation.group);
```

Comment:
I suggest this should use `store.getGroup` via the `useStore` helper rather than `groups.get(<id>)` which is a proxy for that function to ensure that the component re-renders if the group changes.